inference-engine/ie_bridges/python/install/wheel/CMakeLists.txt

```diff
@@ -0,0 +1,98 @@
+# Copyright (C) 2018-2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set(WHEEL_PACKAGE_NAME "openvino" CACHE STRING "Name of the package")
+set(WHEEL_LICENCE_TYPE "Proprietary - Intel" CACHE STRING "License for the package")
+set(WHEEL_AUTOR "Intel Corporation" CACHE STRING "Package author’s name")
+set(WHEEL_AUTOR_EMAIL "openvino_pushbot@intel.com" CACHE STRING "Email address of the package author")
+set(WHEEL_DESC "Inference Engine Python* API" CACHE STRING "Short, summary description of the package")
+set(WHEEL_VERSION "0.0.0" CACHE STRING "Version of this release")
+set(SETUP_PY_IN "${CMAKE_CURRENT_SOURCE_DIR}/setup.py.in")
+set(SETUP_PY    "${CMAKE_CURRENT_BINARY_DIR}/setup.py")
+set(LIBS_CPACK_PATH "libs")
+
+# set version using CI value
+if (WHEEL_VERSION STREQUAL  "0.0.0")
+    string(REGEX MATCH "([0-9?]+)\\.([0-9?]+)\\.([0-9?]+)" WHEEL_VERSION "${CI_BUILD_NUMBER}")
+endif()
+
+# runtime libs
+set(LIB_DIRS    ${CMAKE_INSTALL_PREFIX}/${PYTHON_BRIDGE_CPACK_PATH}/${LIBS_CPACK_PATH})
+# python bindings
+set(BINDINGS_DIR ${CMAKE_INSTALL_PREFIX}/${PYTHON_BRIDGE_CPACK_PATH})
+
+# prepare setup.py
+configure_file(${SETUP_PY_IN} ${SETUP_PY})
```

Comment:
Why not to have just Python script with arguments?

Reply:
Just to simplify usage, we are going to use it in our CI too